src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +17,8 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		DrawerFeature.getDrawerController().clearPanel();
+		if(e.getActionCommand().equals("Figure Joe 2"))FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		else FiguresJoe.figureScript1(driverManager.getCurrentDriver());
```

Comment:
Menu action should not depend on its caption. Enum or at least constants should be considered.